solidity/random-beacon/test/RandomBeacon.Parameters.test.ts

```diff
@@ -216,6 +218,7 @@ describe("RandomBeacon - Parameters", () => {
             newGroupCreationFrequency,
```

Comment:
In the `RandomBeacon.Parameters.test.ts` file, in the "updateGroupCreationParameters" describe, in the "when the caller is the governance", we are missing test checking for update of `dkgResultChallengeExtraGas`.
So we need to add 
```
it("should update the DKG result challenge extra gas", async () => {
...
})
```

Reply:
0973e854597b3195a60b4303f2378378784e2692